sdk/personalizer/test-resources.json

```diff
@@ -0,0 +1,87 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+		"baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+		},
+		"location": {
+            "type": "string",
+            "defaultValue": "centraluseuap",
+            "metadata": {
+                "description": "The location of the resource."
+            }
+		}
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.CognitiveServices/accounts",
+            "apiVersion": "2021-04-30",
+            "name": "[concat(parameters('baseName'), 'SingleSlot')]",
```

Comment:
Tip: define the names as variables you can use throughout to avoid getting out of sync (here and your outputs). See sdk/keyvault/test-resources.ps1 for good examples.